custom_components/hubitat/fan.py

```diff
@@ -0,0 +1,77 @@
+"""Support for Hubitat fans."""
+
+from logging import getLogger
+import re
+from typing import Any, List, Optional
+
+from hubitatmaker import (
+    ATTR_SWITCH,
+    ATTR_SPEED,
+    ATTR_VALUES,
```

Comment:
It doesn't look like this (`ATTR_VALUES`) is used anywhere.

Reply:
(Which is fine -- we don't need it since `values` is an object property rather than a dictionary key.)

Reply:
Ah, you're right.  It uses the `values` property now.